app/Console/Commands/ResetAppCommand.php

```diff
@@ -0,0 +1,71 @@
+<?php namespace VotingApp\Console\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+
+class ResetAppCommand extends Command {
+
+	/**
+	 * The console command name.
+	 *
+	 * @var string
+	 */
+	protected $name = 'reset:app';
+
+	/**
+	 * The console command description.
+	 *
+	 * @var string
+	 */
+	protected $description = 'Reset the app for a new year.';
+
+	/**
+	 * Create a new command instance.
+	 *
+	 * @return void
+	 */
+	public function __construct()
+	{
+		parent::__construct();
+	}
+
+	/**
+	 * Execute the console command.
+	 *
+	 * @return mixed
+	 */
+	public function fire()
+	{
+		//remove non-admin users, password reminders, votes, and winners
+		\DB::table('votes')->delete();
```

Comment:
Since we're using this a couple of times, I'd recommend either importing the facade `use DB;` at the top of this file so you don't need to give it the "global scope" backslash each time, or using the `app()` helper, e.g. `app('db')->table('votes')->delete()`.